test/unit/typography/font_loading.js

```diff
@@ -59,7 +59,7 @@ suite('Fonts outside preload()', function () {
 
     test('should call success-callback with object when font loads',
       function (done) {
-        p5Inst.loadFont('manual-test-examples/p5.Font/acmesa.ttf',
+        p5Inst.loadFont('../manual-test-examples/p5.Font/acmesa.ttf',
```

Comment:
I think we can put these back on where they were before, since the modules' test folder is removed now. What say?